### DISCUSSION TITLE
Профайлы и рабочие группы

### DISCUSSION THREAD

**Discussion by shamilnn** [fhir-ru/core#39] [General]
Для оптимизации работы над профайлами предлагаю создать список FHIR R4/R5(?) ресурсов которые собираемся профилировать, разбить их на логические модули (группы) и создать рабочие группы.
Предпочтение ресурсам с [maturity level](http://hl7.org/fhir/versions.html#maturity ) 3 и выше. 

Например (после ресурса его текущий maturity level):

=== **Base** ===
- Bundle N
- Composition 2

???

=== **Entity** ===
- Patient N
- Practitioner 3
- Organization 3
- Location 3
- PractitionerRole 2

???

=== **Clinical** ===
- Observation N
- Condition 3
- Procedure 3
- AllergyIntolerance  3
- Encounter 2

???

=== **Medication / Immunization** ===
- Immunization 3
- Medication 3
- MedicationRequest 3
- MedicationStatement 3 (FHIR R4)  / MedicationUsage 3 (FHIR R5)

???

=== **Financial** ===
???

=== **Misc** ===
- DiagnosticReport 3
- DocumentReference 3
- Goal 2
- Device 2

???

**Comment by Vrodvi** [comment]
Мне кажется не менее важным определении требований к ключевым типам данных, которые будут повсеместно использоваться во всех обозначенных ресурсах:
[HumanName](https://hl7.org/fhir/R4/datatypes.html#humanname)
[Address](https://hl7.org/fhir/R4/datatypes.html#address)
[Identifier](https://hl7.org/fhir/R4/datatypes.html#identifier)
[CodeableConcept](https://hl7.org/fhir/R4/datatypes.html#codeableconcept)

**Comment by Vrodvi** [comment]
["ООО МираМедикс"](https://miramedix.ru/) выражает заинтересованность в формировании рабочей группы по структуре диагнозов (ресурсы Condition и сопутствующие).
Заведена соответствующая карточка для обсуждения: https://github.com/fhir-ru/core/discussions/47
